version.c

```diff
@@ -7,7 +7,11 @@
 const char git_version_string[] = GIT_VERSION;
 const char git_built_from_commit_string[] = GIT_BUILT_FROM_COMMIT;
 
-static void strbuf_sanitize(struct strbuf *buf)
+/*
+ * Trim and replace each character with ascii code below 32 or above
+ * 127 (included) using a dot '.' character.
```

Comment:
It might be good idea to add something like:

TODO: ensure consecutive non-printable characters are only replaced once

Reply:
I think the TODO should be at the end of the comment, not at the beginning.

Reply:
Ohh, I saw the TODO being placed at the begining of the comment in the codebase. I will put it at the bottom.